test/math.js

```diff
@@ -439,6 +439,14 @@ describe('Math', function () {
       expect(Object.is(-0, Math.round(-0.5 + Number.EPSILON / 3.99))).to.equal(true);
       expect(Object.is(-0, Math.round(0 - Number.EPSILON / 3.99))).to.equal(true);
     });
+
+    it('returns 1 / Number.EPSILON + 1 for 1 / Number.EPSILON + 1', function () {
+      expect(Math.round(1 / Number.EPSILON + 1)).to.equal(1 / Number.EPSILON + 1);
```

Comment:
`1 / Number.EPSILON + 1` - is the smallest number 4503599627370497, for which `Math.floor(x + 0.5)` gives wrong result (4503599627370497 + 0.5 === 4503599627370498)
and `2 / Number.EPSILON - 1` is the largest number for which `Math.flor(x + 0.5)` gives wrong result.